SimPEG/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -21,50 +22,59 @@ class BaseIPSimulation(BaseEMSimulation):
 
     eta, etaMap, etaDeriv = props.Invertible("Electrical Chargeability")
 
-    # surveyPair = Survey
+    data_type = properties.StringChoice(
```

Comment:
would this be better suited to a be a property of the receiver?

Reply:
agree. receiver also have a property of `data_type`. had to set in the simulation to trigger DC simulation and computing predicted DC data.